kotlin/internal/jvm/compile.bzl

```diff
@@ -114,7 +114,7 @@ def _jvm_deps(toolchains, friend, deps, runtime_deps = []):
     )
 
 def _java_infos_to_compile_jars(java_infos):
-    return [compile_jar.path for java_info in java_infos for compile_jar in java_info.compile_jars.to_list()]
```

Comment:
I'm falling down the job -- should have called out the depset in the first pass.